### PR TITLE
Create get end point response for single client

### PR DESCRIPTION
**Pull request by ujwalkumar1995**
Hey @VighneshManjrekar 
I have created the PR for the issue  #10 
Please review.

### CODE REVIEW

File: controllers/main.controllers.js [hunk 1]
```diff
@@ -28,7 +28,22 @@ exports.getProject = async (req, res, next) => {
...
+  try{
+    const client = await Client.findById(id);
+    const response = {
+      id: client.id,
+      name: client.name,
+      email: client.email,
+      phone: client.phone
+    }
+    res.json(response);
+    res.status(200);
```
Comment (VighneshManjrekar) [inline controllers/main.controllers.js]: res.status(200).json(`{success: true, data: { //data goes here }`)

File: controllers/main.controllers.js [hunk 2]
```diff
@@ -28,7 +28,22 @@ exports.getProject = async (req, res, next) => {
...
+  }
+  catch(error){
+    console.log(error);
+    res.sendStatus(500);
```
Comment (VighneshManjrekar) [inline controllers/main.controllers.js]: res.status(500).json({`success: false, error: //error message goes here `} )

File: controllers/main.controllers.js (hunk 1 above)
Comment (VighneshManjrekar) [reply controllers/main.controllers.js]: also handle case when invalid/false id is provided

**Review by VighneshManjrekar — CHANGES_REQUESTED** [review]

**Comment by ujwalkumar1995** [comment]
@VighneshManjrekar 
Hey I have made the suggested change. Invalid id case is already being handled by catch block I think.
Do you need some specific kind of error handling?

**Review by VighneshManjrekar — CHANGES_REQUESTED** [review]
Need invalid id(400) and client not found (404) response

**Comment by ujwalkumar1995** [comment]
@VighneshManjrekar 
Have added 400 and 404 error types. Please review.

**Review by VighneshManjrekar — APPROVED** [review]

**Pull request merged**